man/rgbgfx.1

```diff
@@ -378,14 +378,18 @@ The following formats are supported:
 .Lk https://www.adobe.com/devnet-apps/photoshop/fileformatashtml/#50577411_pgfId-1070626 Adobe Photoshop color table .
 .It Cm aco
 .Lk https://www.adobe.com/devnet-apps/photoshop/fileformatashtml/#50577411_pgfId-1055819 Adobe Photoshop color swatch .
-.It Cm gbc
```

Comment:
I put them in alphabetic order; do you have a more specific order in mind?

Reply:
Oh, I saw it as "official formats supported by editors", "unofficial plaintext formats", and then "custom GBC format". Alphabetical is less arbitrary though.